src/ovp_pipeline/entities/aliases.py

```
@@ -0,0 +1,422 @@
+        same thing)
+      * one ``explicit_alias`` row per ``aliases[]`` entry
+    """
+    if not path.exists():
+        return []
+    out: list[EntityAlias] = []
+    # Stream the file line-by-line.  Authors files can grow to a few
+    # MB on busy vaults; loading + splitlines doubles peak memory
+    # for no benefit.
+    with path.open(encoding="utf-8") as f:
+        for line in f:
+            line = line.strip()
```

Ruff (0.15.12)

[warning] `for` loop variable `line` overwritten by assignment target

(PLW2901)